xml/docu_styleguide.language.xml

```diff
@@ -445,6 +451,14 @@
    <guimenu>OK</guimenu> button.</quote> However, complex dialogs may
    require more specific wording.
   </para>
+  <para>
+   Format the label of user interface items in sentence case, even if they are
+   formatted differently in the user interface.
+   For more information about sentence case, see
+   <xref linkend="sec-capitalization-sentence"/>.
+   This is in accordance with the &suse; product style guide at
+   <link xlink:href="https://productbrand.suse.com/writing/conventions-and-rules"/>.
+  </para>
```

Comment:
ISTM that this decision has been reversed now, right?

Reply:
(yes & updated)